genesis/parsing/accountsParser_test.go

```diff
@@ -744,3 +747,11 @@ func TestAccountsParser_GenerateInitialTransactionsVerifyTxsHashes(t *testing.T)
 		assert.Equal(t, tx, v.Transaction)
 	}
 }
+
+func TestAccountsParser_GenesisMintingAddress(t *testing.T) {
+	t.Parallel()
+
+	ap := parsing.NewTestAccountsParser(createMockHexPubkeyConverter())
+	addr := ap.GenesisMintingAddress()
+	assert.Equal(t, hex.EncodeToString([]byte("erd17rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rcqqkhty3")), addr)
```

Comment:
maybe extract minting address into a const since we have it in 3 places in this package

Reply:
[]byte("erd1...") is confusing. maybe use the real bech32 converter or use a hex / dummy address here